libs/features/seotech/seotech.js

```diff
@@ -21,14 +21,45 @@ export async function getVideoObject(url, seotechAPIUrl) {
   return body.videoObject;
 }
 
-export default async function appendVideoObjectScriptTag(url, { createTag, getConfig }) {
+export async function getStructuredData(url, sheetUrl, seotechAPIUrl) {
+  const apiUrl = new URL(seotechAPIUrl);
+  apiUrl.pathname = '/api/v1/web/seotech/getStructuredData';
+  apiUrl.searchParams.set('url', url);
+  if (sheetUrl) {
+    apiUrl.searchParams.set('sheetUrl', sheetUrl);
+  }
+  const resp = await fetch(apiUrl.href, { headers: { 'Content-Type': 'application/json' } });
+  const body = await resp?.json();
+  if (!resp.ok) {
+    throw new Error(`Failed to fetch structured data: ${body?.error}`);
```

Comment:
Should this also contain the `sheetUrl` so the Splunk messages are clear as to which sheet failed?

Reply:
`sheetUrl` is used by the service, so if there is a problem with it then the URL should be returned in `body.error` msg. Will fix api to do this.